internal/pkg/template/env.go

```diff
@@ -108,6 +113,7 @@ type EnvOpts struct {
 
 	LatestVersion      string
 	SerializedManifest string // Serialized manifest used to render the environment template.
+	ForceUpdateID      string
```

Comment:
Maybe make any occurences of `ForceUpdateID` a pointer to reflect that not every deployment is forced? I know "" covers that currently, but it may work better if it's needed to be used in the future to check if it's nil.

O/w LGTM

Reply:
I've been actually torn between whether to make it a pointer! 

If it is a pointer, we'd probably have in `cf.yml`:
```
{{- if .ForceUpdateID}}
DeploymentController:
  Value: {{.ForceUpdateID}}
{{- end}}
```

The UX would (probably) be:
1. I as a user, have been running `copilot env deploy` without the flag ➡️  no `DeploymentController` in my stack
2. One day, I ran `copilot env deploy --force` ➡️  `DeploymentController` shows up ➡️  I guess it makes sense because I used `--force`.
3. I then ran `copilot env deploy` ➡️  `DeploymentController` is still there though the value doesn't change ➡️  Confused! I thought `DeploymentController` only shows up when I `--force`!

The pros is that for people that will never ever use `--force` flag, this crazy output won't ever get added to their template. However I'm worried about 3.

On the other hand, if we keep it as it, without the pointer. The UX would be:
4. I as a user, have been running `copilot env deploy` without the flag ➡️  `DeploymentController` has always been in my stack.
5. One day, I ran `copilot env deploy --force` ➡️  `DeploymentController` value changes ➡️  I guess it makes sense because I used `--force`.
6. I then ran `copilot env deploy` ➡️  `DeploymentController` is still there though the value doesn't change ➡️  I'm familiar with this output because It's been always there. I guess it just changes value when I run `--force`. 

The cons is that it's going to be there even if the user never uses `--force`. However, it doesn't introduce the "irreversible flip" as described in 3.

Reply:
I can see that concern. Would it then be possible to do something like resetting the ID when `--force` is unused?